AcsfToolsUtils.php

```diff
@@ -33,6 +33,12 @@ public function getSites() {
         if (!isset($sites[$site_details['name']])) {
           $sites[$site_details['name']] = $site_details;
         }
+
+	// Path domains need a trailing slash to be recognized as a drush alias.
+	if (FALSE !== strpos($domain, '/')) {
```

Comment:
can we try and avoid yoda conditions?

https://en.m.wikipedia.org/wiki/Yoda_conditions